src/Features/Core/Completion/CompletionList.cs

```diff
@@ -34,7 +34,7 @@ internal class CompletionItemGroup
         /// </summary>
         public CompletionItem Builder { get; }
 
-        public CompletionItemGroup(IEnumerable<CompletionItem> items, CompletionItem builder = null, bool isExclusive = false)
+        public CompletionList(IEnumerable<CompletionItem> items, CompletionItem builder = null, bool isExclusive = false)
         {
             this.Items = items.ToImmutableArray();
```

Comment:
Is toImmutableArray smart if this is already an ImmutableArray?  Otherwise you're copying again.

Also, i think we could consider tightening this up.  There's danger with IEnumerables.